src/components/Navbar.tsx

```diff
@@ -8,15 +11,22 @@ export const Navbar = () => {
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">Home</a>
+          <NavLink
+            to="/"
+            className={({ isActive }) => classNames('navbar-item',
+              { 'has-background-grey-lighter': isActive })}
+          >
+            Home
+          </NavLink>
 
-          <a
+          <NavLink
+            to="people"
```

Comment:
It's better to use an absolute path for the NavLink `to` property. Add a leading slash to make it `/people` instead of `people`. 🛣️

Reply:
fixed